test/parallel/test-dgram-send-empty-buffer.js

```diff
@@ -1,26 +1,25 @@
 'use strict';
 const common = require('../common');
-const dgram = require('dgram');
 
 if (common.isOSX) {
   common.skip('because of 17894467 Apple bug');
   return;
 }
 
+const dgram = require('dgram');
+
 const client = dgram.createSocket('udp4');
 
-client.bind(0, function() {
+client.bind(0, common.mustCall(function() {
   const port = this.address().port;
 
   client.on('message', common.mustCall(function onMessage(buffer, bytes) {
-    clearTimeout(timer);
+    clearInterval(interval);
     client.close();
   }));
```

Comment:
maybe check that buffer is empty. Also s/bytes/info or even removing the 2nd argument if we're doing nothing with it